src/routes/accommodationRoute.js

```diff
@@ -1,11 +1,11 @@
 import express from 'express';
 import fileUpload from 'express-fileupload';
 import path from 'path';
-import accommodationCont from '../controllers/accommodationController';
-import accMidd from '../middlewares/accommodationMiddleware';
-import userMidd from '../middlewares/userMiddlware';
-import valid from '../validation';
-import roles from '../middlewares/rolesMiddlewares';
+import accommodationsController from '../controllers/accommodations.controller';
```

Comment:
Let's use `AccommodationService` here, as well as `AccommodationMiddlware` and `AccommodationsController`

Reply:
resolved